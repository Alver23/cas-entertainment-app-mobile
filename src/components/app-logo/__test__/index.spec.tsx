// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Image } from 'react-native';

// Components
import { AppLogoView } from '../index';

const MemoizeComponent = AppLogoView.type;

describe('AppLogoView component', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<MemoizeComponent />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly', () => {
    expect(component.find(Image).exists()).toBeTruthy();
  });
});
