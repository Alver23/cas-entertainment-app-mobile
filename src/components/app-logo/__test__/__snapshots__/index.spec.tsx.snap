// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogoView component should save a snapshot of the component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/app-logo.png",
      }
    }
  />
</View>
`;
